modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/constants/CommerceOrderFragmentFDSNames.java

```diff
@@ -26,6 +26,10 @@ public class CommerceOrderFragmentFDSNames {
 		CommerceFragmentRendererKeys.ORDER_RETURNS_DATA_SET +
 			"-placedOrderReturns";
 
+	public static final String PLACED_ORDER_SHIPMENTS =
+		CommerceFragmentRendererKeys.ORDER_RETURNS_DATA_SET +
```

Comment:
Why use the root string of `ORDER_RETURNS_DATA_SET`?
```
public static final String ORDER_RETURNS_DATA_SET =
		"com_liferay_commerce_order_content_web_internal_fragment_renderer_" +
			"OrderReturnsDataSetFragmentRenderer";
```

I think this should be either a new key or a modifier of `ORDERS_DATA_SET`.